setup_environment.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+###################################################################
+# The purpose of this script is to download, install
+# and configure all files and programs required to
+# take part in developing ActivitySync (Android only).
+###################################################################
+
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+NOCOLOR='\033[0m'
+
+# update package lists from repositories
+sudo apt-get update
+
+# install git
+git --version 2>&1 >/dev/null
+GIT_IS_AVAILABLE=$?
+if [ $GIT_IS_AVAILABLE -eq 0 ]; then
+    echo "Skipping git installation (already exists)."
+else
+    sudo apt-get install git
+fi
+
+# keep all work inside /tmp directory
+cd /tmp/
+
+# create directory for java
+sudo mkdir /usr/lib/jvm
+
+# download, install and configure java7 jdk
+sudo wget --no-check-certificate --no-cookies --header "Cookie: oraclelicense=accept-securebackup-cookie" http://download.oracle.com/otn-pub/java/jdk/7u79-b15/jdk-7u79-linux-x64.tar.gz
```

Comment:
I do not understand. Why root access is needed to get a file using wget and store it in /tmp ?